cvmfs/swissknife_lease.cc

```diff
@@ -0,0 +1,141 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "swissknife_lease.h"
+
+#include <fstream>
```

Comment:
Is it a larger effort to get rid of the C++ file and string streams? Everywhere else the C I/O + cvmfs helper functions from util/string.cc and util/posix.cc are used.